app/src/main/java/gov/hhs/cdc/trustedintermediary/OpenApi.java

```diff
@@ -21,6 +21,7 @@ public static OpenApi getInstance() {
 
     public String generateApiDocumentation(final Set<String> openApiSpecifications) {
         openApiSpecifications.add(getBaselineDocumentation());
+        openApiSpecifications.add(getEtorDocumentation());
 
```

Comment:
I think we can refactor this better. Maybe we can create a method that adds all of the existing documentation. It feels like `getBaselineDocumentation()` and `getEtorDocumentation()` do a similar job. What are your thoughts?

Reply:
Did not need this line of code as the Etor documentation comes in via the parameter `openApiSpecifications`